docs/src/main/asciidoc/_cloud/References.adoc

```diff
@@ -0,0 +1,21 @@
+# References
```

Comment:
Is there a link to this page from the Getting Started doc?

Reply:
Nope, there isn't. I'll try to create one